var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -516,6 +546,19 @@ def cmake_args(self):
                 + f'/{self.spec["nvhpc"].version}/math_libs'
             )
 
+        if "+heffte" in self.spec:
+            options.append("-DGMX_USE_HEFFTE=on")
+            options.append(f'-DHeffte_ROOT={self.spec["heffte"].prefix}')
+
+        if "+intel-data-center-gpu-max" in self.spec:
```

Comment:
This matches https://manual.gromacs.org/documentation/current/install-guide/index.html#sycl-gpu-acceleration-for-intel-gpus

Reply:
Thanks!